src/core/middlewares/rtkQueryError.ts

```diff
@@ -1,17 +1,36 @@
 import type { Middleware, MiddlewareAPI } from '@reduxjs/toolkit';
 import { isRejectedWithValue } from '@reduxjs/toolkit';
+import { forEach } from 'lodash';
 
 import toast from '@/components/Toast';
 import Events from '../events';
 
+type ProblemDetails = { title: string; status: number; type: string; errors: Record<string, string[]>; traceId: string; };
```

Comment:
It's named `ProblemDetails` since that's the name of the type on the server side.